cves/CVE-2014-0480.yml

```diff
@@ -15,24 +15,24 @@ curated_instructions: |
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
-curated: false
+curated: true
 reported_instructions: |
   What date was the vulnerability reported to the security team? Look at the
   security bulletins and bug reports. It is not necessarily the same day that the
   CVE was created.  Leave blank if no date is given.
   Please enter your date in YYYY-MM-DD format.
-reported_date:
+reported_date: 2013-12-19
 announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE date. A good
   source for this is Chrome's Stable Release Channel
   (https://chromereleases.googleblog.com/).
   Please enter your date in YYYY-MM-DD format.
-announced_date:
+announced_date: 2014-08-20
```

Comment:
wow it took a while for this to be announced after being reported

Reply:
Yep.  And they waited another month after creating the fix to announce it publicly.